### PR TITLE
Send kafka tracking messages

### PR DESCRIPTION
**Pull request by guptad2**

### CODE REVIEW

File: cache/models/samples.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ const sampleNameSeparator = '|';
 const logger = require('@salesforce/refocus-logging-client');
 const featureToggles = require('feature-toggles');
 const config = require('../../config');
+const tracker = require('../../kafkaTracking');
```
Comment (jgraff2) [inline cache/models/samples.js]: Is this used?

Comment (guptad2) [reply cache/models/samples.js]: Just fixed it

**Comment by iamigo** [comment]
All the travis failed tests

**Comment by guptad2** [comment]
@iamigo just fixed, there was a typo

**Pull request merged**